cssjson.js

```diff
@@ -58,6 +58,10 @@ var CSSJSON = new function () {
         return typeof x == 'undefined' || x.length == 0 || x == null;
     };
 
+	var isCssJson = function (node) {
+		return !isEmpty(node) ? (node.attributes && node.children) : false;
+	}

```

Comment:
Could you please fix the indentation for this and the toHead() method? I used 4 spaces.